_layouts/project.html

```diff
@@ -2,8 +2,16 @@
 layout: default
 ---
 
-{% if page['Practice Area'].size > 0 %}
-	{% assign header_title = page['Practice Area'] | first %}
+{% assign foundPracticeAreas = site.newEmptyArray %}
```

Comment:
@tehandyb Is this change needed since you updated how the project practice area lists are built?

Reply:
Yeah this thing is still needed